src/client/apps/articles_list/components/articles_list.jsx

```diff
@@ -99,16 +100,22 @@ export class ArticlesList extends Component {
     // TODO: remove jQuery
     if ($('.filter-search__input').val()) { return }
```

Comment:
re: the loading spinner TODO, we have an action for this in Redux -- I believe it is scoped to the edit page by selector, but could probably be adjusted to work here: https://github.com/artsy/positron/blob/master/src/client/actions/editActions.js#L338

Reply:
It actually uses jquery too but could be refactored pretty easily into react.